### PR TITLE
CDAP-12079 Reduce kafka request timeout to 15 seconds, from the default of 305.

### PR DESCRIPTION
**Pull request by anwar6953**
https://issues.cask.co/browse/CDAP-12079
Reduce kafka request timeout to 15 seconds, from the default of 305 seconds.

### CODE REVIEW

File: wrangler-service/src/main/java/co/cask/wrangler/service/kafka/KafkaService.java [hunk 1]
```diff
@@ -124,14 +124,12 @@ public void test(HttpServiceRequest request, HttpServiceResponder responder) {
...
-      KafkaConsumer<String, String> consumer = new KafkaConsumer<>(props);
-      try {
+      try (KafkaConsumer<String, String> consumer = new KafkaConsumer<>(props)) {
         consumer.listTopics();
-      } finally {
-        consumer.close();
       }
       ServiceUtils.success(responder, String.format("Successfully connected to Kafka at %s", config.getConnection()));
     } catch (Exception e) {
+      e.printStackTrace();
```
Comment (bdmogal) [inline wrangler-service/src/main/java/co/cask/wrangler/service/kafka/KafkaService.java]: remove?

**Review by bdmogal — APPROVED** [review]
tiny comment, then lgtm, presume tested on UI

File: wrangler-service/src/main/java/co/cask/wrangler/service/kafka/KafkaService.java (hunk 1 above)
Comment (anwar6953) [reply wrangler-service/src/main/java/co/cask/wrangler/service/kafka/KafkaService.java]: oops... I don't recall doing this.

**Comment by anwar6953** [comment]
Changed base branch to release/3.0

**Pull request merged**